### DISCUSSION TITLE
Grand Prix Radio Replays

### DISCUSSION THREAD

**Discussion by dernl** [f1multiviewer/issue-tracker#125] [Feature requests]
Add a way to access Grand Prix Radio during a replay. For example make an arrow next to the "Open Grand Prix Radio" button which goes to an archive with old sessions.

**Comment by supersnellehenk** [comment]
They're listed here https://omny.fm/shows/race-gemist or here https://grandprixradio.nl/updates/race-gemist-f1-commentaar-olav-mol-terug-te-luisteren-1.

The disclaimer basically states they you need written approval from GPRadio and mentioning of the source to be able to list it.

**Reply by f1multiviewer** [reply]
Yeah I've sent them an email but their email doesn't even work, so I'll just assume they're OK with it and link them back.

**Reply by dernl** [reply]
@supersnellehenk, yes, but it's only  the race, which is a bit annoying, since I would also like to be able to do the same when I watch a replay of the qualifying
